src/erlcloud_kinesis.erl

```diff
@@ -312,25 +312,25 @@ get_shard_iterator(StreamName, ShardId, ShardIteratorType, StartingSequenceNumbe
 %%------------------------------------------------------------------------------
 
 -spec get_records/1 :: (string()) -> proplist().
-
 get_records(ShardIterator) ->
-  Json = [{<<"ShardIterator">>, ShardIterator}],
-  get_normalized_records(default_config(), Json).
-
--spec get_records/2 :: (string(), 1..100 | aws_config()) -> proplist().
+    Json = [{<<"ShardIterator">>, ShardIterator}],
+    get_normalized_records(default_config(), Json).
 
+-spec get_records/2 :: (string(), 1..10000 | aws_config()) -> proplist().
 get_records(ShardIterator, Config) when is_record(Config, aws_config) ->
-  Json = [{<<"ShardIterator">>, ShardIterator}],
-  erlcloud_kinesis_impl:request(Config, "Kinesis_20131202.GetRecords", Json);
-get_records(ShardIterator, Limit) when is_integer(Limit), Limit > 0, Limit =< 100 ->
-  Json = [{<<"ShardIterator">>, ShardIterator}, {<<"Limit">>, Limit}],
-  get_normalized_records(default_config(), Json).
-
--spec get_records/3 :: (string(), 1..100, aws_config()) -> proplist().
-
-get_records(ShardIterator, Limit, Config) when is_record(Config, aws_config), is_integer(Limit), Limit > 0, Limit =< 100 ->
-  Json = [{<<"ShardIterator">>, ShardIterator}, {<<"Limit">>, Limit}],
-  get_normalized_records(Config, Json).
+    Json = [{<<"ShardIterator">>, ShardIterator}],
+    get_normalized_records(Config, Json);
+get_records(ShardIterator, Limit)
+  when is_integer(Limit), Limit > 0, Limit =< 10000 ->
+    get_records(ShardIterator, Limit, default_config()).
+
+-spec get_records/3 :: (string(), 1..10000, aws_config()) -> proplist().
```

Comment:
@motobob 
According to AWS specification:
Limit
The maximum number of records to return. Specify a value of up to 10,000. If you specify a value that is greater than 10,000, GetRecords throws InvalidArgumentException.
    Type: Number
    Valid range: Minimum value of 1. Maximum value of 10000.
    Required: No 

get_records should provide an ability to return up to 10000 records.